src/pages/my-policies/my-policies.tsx

```diff
@@ -0,0 +1,86 @@
+import { FormEventHandler } from 'react';
```

Comment:
This was also extracted from the previous `src/pages/policies/policies.tsx` file